src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -32,9 +32,15 @@ enum ErrorType {
 	BAD_REQUEST
 }
 
+enum ErrorCode {
```

Comment:
nit: Let's use ErrorFlags instead of ErrorCode as we arrived at during our catchup? Will map the naming of the error value to how we would like to use them `if flags&RESOURCE_EXHAUSTED { ... }`.